deduplication.Rmd

```diff
@@ -0,0 +1,296 @@
+---
```

Comment:
Didn't go through this file since the previous one looked like it had both SAX and cross-correlation in it.  Do I need to review this one also?